jackal_control/launch/control.launch

```diff
@@ -1,10 +1,10 @@
 <launch>
-  <rosparam command="load" file="$(find jackal_control)/config/control.yaml" />
+  <rosparam file="$(find jackal_control)/config/control.yaml" command="load"/>
 
   <node name="controller_spawner" pkg="controller_manager" type="spawner"
         args="jackal_joint_publisher jackal_velocity_controller --shutdown-timeout 1" />
-  
-  <node pkg="robot_localization" type="ekf_localization_node" name="ekf_localization" /> 
+
+  <include file="$(find jackal_control)/launch/robot_localization.launch"/>
```

Comment:
@mikepurvis Please let me know if you prefer `control.launch` w/o the nested `robot_localization.launch`; in that case, I'll include it on the two other launch files.

If you want, we can also close this and I'll create my own separate `robot_localization.launch`.